src/EditorFeatures/Core/BraceMatching/BraceHighlightTag.cs

```diff
@@ -4,9 +4,10 @@
 
 #nullable disable
 
+using Microsoft;
```

Comment:
what is in this namespace?

Reply:
Shouldn't have been added. Bug in refactoring: https://github.com/dotnet/roslyn/issues/59899